app/templates/main/poc-search.html

```diff
@@ -34,28 +34,69 @@ <h3 class="govuk-heading-s govuk-!-font-weight-bold">Search for digital records<
             <h2 class="govuk-heading-m">{{ num_records_found }} record(s) found</h2>
-            {% if num_records_found > 0 %}
-            <table class="govuk-table">
-              <thead class="govuk-table__head">
-              <tr class="govuk-table__row govuk-!-font-size-14">
-                <th scope="col" class="govuk-table__header govuk-!-font-weight-bold">Transferring body</th>
-                <th scope="col" class="govuk-table__header govuk-!-font-weight-bold">Series</th>
-                <th scope="col" class="govuk-table__header govuk-!-font-weight-bold">Consignment reference</th>
-                <th scope="col" class="govuk-table__header govuk-!-font-weight-bold">File Name</th>
-              </tr>
-              </thead>
-              <tbody class="govuk-table__body">
-              {% for record in results %}
-              <tr class="govuk-table__row">
-                <td class="govuk-table__cell govuk-body govuk-!-font-size-14">{{ record["transferring_body"] }}</td>
-                <td class="govuk-table__cell govuk-body govuk-!-font-size-14">{{ record["series"] }}</td>
-                <td class="govuk-table__cell govuk-body govuk-!-font-size-14">{{ record["consignment_reference"] }}</td>
-                <td class="govuk-table__cell govuk-body govuk-!-font-size-14"><a href="{{ url_for('main.record', record_id=record['file_id'])}}">{{ record["file_name"] }}</href></td>
-              </tr>
-              {% endfor %}
-              </tbody>
-            </table>
-            {% endif %}
+              {% if num_records_found > 0 %}
+                <table class="govuk-table">
+                  <thead class="govuk-table__head">
+                  <tr class="govuk-table__row govuk-!-font-size-14">
+                    <th scope="col" class="govuk-table__header govuk-!-font-weight-bold">Transferring body</th>
+                    <th scope="col" class="govuk-table__header govuk-!-font-weight-bold">Series</th>
+                    <th scope="col" class="govuk-table__header govuk-!-font-weight-bold">Consignment reference</th>
+                    <th scope="col" class="govuk-table__header govuk-!-font-weight-bold">File name</th>
+                  </tr>
+                  </thead>
+                  <tbody class="govuk-table__body">
+                  {% for row in results %}
+                  <tr class="govuk-table__row">
+                    <td class="govuk-table__cell govuk-body govuk-!-font-size-14">{{ row[0] }}</td>
+                    <td class="govuk-table__cell govuk-body govuk-!-font-size-14">{{ row[1] }}</td>
+                    <td class="govuk-table__cell govuk-body govuk-!-font-size-14">{{ row[2] }}</td>
+                    <td class="govuk-table__cell govuk-body govuk-!-font-size-14">{{ row[3] }}</td>
+                  </tr>
+                  {% endfor %}
+                  </tbody>
+                </table>
+                <nav class="govuk-pagination" role="navigation" aria-label="Pagination">
```

Comment:
can we extract this whole pagination nav block into its own template which we can include in here using

https://flask.palletsprojects.com/en/1.1.x/patterns/templateinheritance/

Reply:
extracted pagination object in separate template file pagination.html and used include tag to reference the pagination template.